### ISSUE TITLE
Provide detailed network interface/status in log file

### ISSUE THREAD

**Issue by Martin-Weiss** [SUSE/suse-migration-services#84]
During some issues with the network setup I realized that it is a bit difficult to find out the root cause for sub-optimal network setup while in the temporary system.

Maybe a simple "ip a" / "routes -n" and "cat /etc/resolv.conf" could be added to the /var/log/distro_migration.log after a successful network setup?

**Comment by jesusbv** [comment]
Hi @Martin-Weiss, thank you for the suggestion.
What issues did you face during the network setup?

**Comment by schaefi** [comment]
@jesusbv Martin was working on a state that didn't had the network fixes I made. So the race condition for the network.online target we discussed applied to his image. I suggested to switch to the one we have released and that solved the network problem.

However I agree if there is trouble with the network it would be hard to find the root cause. I'm not so convinced that ip or dns information would have helped to find the issue though.

We should also be careful what information we populate into the log file. ```ip a``` lists all interfaces, metrics and partial network capabilities.

If we add more information it imho should be explicit and connected to a certain type of defect. As we from this report only have a vague information and the root cause was a pre-production image I'd like to ask @Martin-Weiss what information exactly would be helpful to identify network problems ?

The log already contains the long status version from all services, that includes the network service as well. Actually if the service gets called and there is a problem you should get good data

Thoughts ?

**Comment by jesusbv** [comment]
Agreed. I guessed the problem could be gone after your network fix, that is why I asked to know what issues and also to find out what information could be relevant that is no there yet.

**Comment by Martin-Weiss** [comment]
Yes - the last network problem is fixed - but if the next one happens it would be really nice to have a debug option or some standard logging for the network configuration. And especially with the SES 6 upgrades where we expect to have complicated network setups it would be really good to have more network logging information or at least a further debug option for the network.

Based on the existing logging the first problem I understood during trouble shooting was that DNS/resolv.conf was not working because zypper error was "can not resolv name.." - only after command line access on the live system I realized that there was no network available at all.
While reading the current log from the top I just had a single line about the network setup without any error or details:

`[INFO    ]: 13:55:27 | Running setup host network service`

I also did not realize that eth0/eth1 were missing in the wicked.service output.
(I did not have a working example to compare and did not know that later in the logfile I find info corresponding to the early lines in the log)

As basic information it would be nice to have the network details / results that are active after the network start in the live system (network interfaces, bonds, vlans). There is so much zypper output that having some network details (Interfaces configured, IPs v4+v6, netmasks, gateways, dns config, hosts file) should not be a big problem IMHO ;-).
